src/sys/wait.rs

```diff
@@ -145,27 +145,34 @@ mod status {
           target_os = "netbsd"))]
 mod status {
     use sys::signal::Signal;
+    use libc::c_int;
```

Comment:
Why is it necessary to change all of these types?

Reply:
I agree with @asomers here, please remove this changes from this PR.